skt/runner.py

```diff
@@ -485,6 +485,29 @@ def __jobsubmit(self, xml):
 
     def run(self, url, release, wait=False, host=None, uid="",
             arch=platform.machine(), reschedule=True):
+        """
+        Main method for testing in Beaker.
+
+        Args:
+            url:
+            release:    NVR of the tested kernel.
+            wait:       False if skt should exit after submitting the jobs,
+                        True if it should wait for them to finish.
+            host:       Force testing on a machine with provided hostname.
+            uid:        Username jobs should be submitted under. Can be
+                        retrieved by running `bkr whoami`.
+            arch:       Architecture of the machine the tests should run on.
+                        Defaults to architecture of the current machine skt is
+                        running on if not specified.
+            reschedule: True to try to rule out infrastructure / host-specific
+                        failures by resubmitting the job (on both the same and
+                        different host), False otherwise.
+
+        Returns:
+            ret: 0 if everything passed
```

Comment:
Google Python Style Guide doesn't seem to require documenting variables used internally to store return values. And why would the function user need to know?

Is there some reason/purpose I'm missing, or could you perhaps remove it?

Reply:
That's the name of the variable being returned, and explanation of possible values. I can remove the name if that's what's confusing